src/main/interface/msp.c

```diff
@@ -1224,8 +1224,8 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0); // reserved
         sbufWriteU8(dst, currentPidProfile->vbatPidCompensation);
         sbufWriteU8(dst, currentPidProfile->setpointRelaxRatio);
-        sbufWriteU16(dst, currentPidProfile->dtermSetpointWeight);
-//        sbufWriteU8(dst, 0); // reserved
+        sbufWriteU8(dst, currentPidProfile->dtermSetpointWeight);
```

Comment:
this should be `sbufWriteU8(dst, MIN(currentPidProfile->dtermSetpointWeight, 255))` so that legacy apps show '255' when the real value is higher than 255.

Reply:
You are right. I will try to change it.